app/build.gradle

```diff
@@ -88,6 +88,10 @@ dependencies {
     implementation 'com.squareup.moshi:moshi:1.14.0'
```

Comment:
Please remove old unused moshi libraries, they still takes space storage in the app 😉